index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1 class="header_name">Nicholas</h1>
+      <nav class="header_menu">
+        <a href=""><img src="image/Union.png" alt="Union image" /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <h2 class="title_1">I’m Nicholas Glad to see you!</h2>
```

Comment:
Kindly make this an `h1` tag, since this is the most important heading on this page.  Also, consider increasing the margin between the top of the `section` and this heading. If you follow the Figma template you will notice there is a significant margin between the top of the section and the heading.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/212344347-be1773b8-9ef5-4dbe-b1e7-90a71227b561.png) | ![image](https://user-images.githubusercontent.com/33268927/212344703-66aae3eb-18b4-4484-a301-4c231bd6132c.png) |